packages/playground/src/components/calculate-divergence-bar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Text } from '@harnessio/canary'
+import { cn } from '@harnessio/canary'
+
+interface CalDivergenceProps {
+  behindAhead: {
+    behind?: number
+    ahead?: number
+  }
+  className?: string
+}
+
+export const CalculateDivergenceBar = ({ behindAhead, className }: CalDivergenceProps) => {
```

Comment:
Including the verb calculate in the component name could be anti-pattern. I hate to nitpick but this component might be better named DivergenceGauge?

Reply:
I agree